pkg/kubelet/kubelet.go

```diff
@@ -757,6 +763,16 @@ type Kubelet struct {
 
 	// handlers called during the tryUpdateNodeStatus cycle
 	setNodeStatusFuncs []func(*api.Node) error
+
+	// TODO: think about moving this to be centralized in PodWorkers in follow-on.
+	// the list of handlers to call during pod admission.
+	lifecycle.PodAdmitHandlers
```

Comment:
It is unfortunate that `syncloop` for a pod is part of kubelet :( Since you are mentioning TODOs for the future, it will be useful to move all the pod sync logic into the pod workers.